src/pages/en/guides/markdown-content.md

```diff
@@ -206,9 +208,9 @@ Here is my counter component, working in MDX:
 <ReactCounter client:load />
 ```
 
-#### Assigning Custom Components to HTML elements
+#### Assigning Custom Components to HTML elements in MDX
 
-MDX allows some common HTML elements to be written either with Markdown syntax or with JSX expressions. For example, `*italics*` for emphasis can also be written inside a JSX expression as `<em>italics</em>`. Both are converted to HTML elements.
+MDX allows some common HTML elements to be written either with Markdown syntax or with JSX expressions. For example, `*italics*` for emphasis can also be written inside a JSX expression as `<em>italics</em>`.
 
 It is also possible to map these Markdown symbols to `.astro` components instead of their standard HTML elements. This allows you to write in standard Markdown syntax without adding components in your Markdown content.
```

Comment:
Instead, I’d probably start this section with this paragraph, reworked a bit like this maybe (removed `.astro` because I think you should be able to pass any framework component you’re using in this context):

```suggestion
With MDX, you can map Markdown syntax to custom components instead of their standard HTML elements. This allows you to write in standard Markdown syntax, but apply special component styling to selected elements.
```

Reply:
I might also rework the example to instead of saying

> Now, when you type `>`, you can render a custom `<Blockquote />` component instead of a regular `<blockquote>` HTML element:

We actually show the quote inside `about.mdx`

```mdx
import Blockquote from `../components/Blockquote.astro';
export const components = {blockquote: Blockquote}

> This quote will be wrapped by our blockquote component
```